tests/cypress/e2e/features/shortcuts.js

```diff
@@ -155,8 +155,6 @@ context('Customizable Shortcuts', () => {
 
     describe('Saving, Clearing and Restoring to Default', () => {
         it('Saving shortcuts and checking if they persist', () => {
-            cy.openSettings();
-            cy.saveSettings();
```

Comment:
Probably the remaining part of this test should be moved to the previous describe, as ctrl+space is assigned there.

Reply:
Moved